ext/MitsubaLoader/CElementMatrix.cpp

```diff
@@ -0,0 +1,83 @@
+#include "CElementMatrix.h"
+
+#include "ParserUtil.h"
+#include "CSimpleElement.h"
+
+namespace irr { namespace ext { namespace MitsubaLoader {
+
+
+bool CElementMatrix::processAttributes(const char** _atts)
+{
+
+	switch (type)
+	{
+	case CElementMatrix::Type::ARBITRARY:
+		for (int i = 0; _atts[i]; i += 2)
```

Comment:
shouldn't it always be `_atts[2]==nullptr` and `_atts[0] && _atts[1]` ?

Reply:
if .xml file structure is correct, then _atts[0] is always "value" and _atts[1] is a string consisting of 16 floats

Reply:
there's a thing called valiation, someone might be trying to feed the loader a bad .XML and the loader should not crash because of that